pkg/provider/azure_standard.go

```diff
@@ -1086,3 +1086,8 @@ func (as *availabilitySet) GetNodeNameByIPConfigurationID(ipConfigurationID stri
 	}
 	return vmName, strings.ToLower(asName), nil
 }
+
+// GetNodeCIDRMaskByProviderID returns the node CIDR subnet mask by provider ID.
+func (as *availabilitySet) GetNodeCIDRMasksByProviderID(providerID string) (int, int, error) {
+	return 0, 0, cloudprovider.NotImplemented
```

Comment:
we could use default mask size for VMAS nodes as a start point.

Reply:
yes, I will do that